### ISSUE TITLE
add oauth client/secret to dbt_cloud_connection

### ISSUE THREAD

**Issue by trunsky** [GtheSheep/terraform-provider-dbt-cloud#69]
Would it be possible to add `oauth_client_id` and  `oauth_client_secret` to the dbt_cloud_connection resource?

**Comment by GtheSheep** [comment]
Yep, giving it a shot right now as it happens, hopefully will update shortly 😺

**Comment by GtheSheep** [comment]
So I can add it but it doesn't seem to appear in the response from query the connection API atm, so I'll add a todo to fix that when it does, should still be ok to manage it in a basic sense though, if you get a chance and could try querying one of your connections that uses Oauth just to check it definitely isn't coming through that would be much appreciated:
```
curl -H '{"Content-Type": "application/json", "Authorization": "Token <api_token>"}' https://cloud.getdbt.com/api/v3/accounts/<account_id>/projects/<project_id>/credentials/ 
```

**Comment by trunsky** [comment]
Just checked - it's definitely not coming through the `/connections` get request

**Comment by GtheSheep** [comment]
Awesome, thanks! I'll merge the PR for it anyway, as it'll allow managing it in CI-CD, just won't be able to detect if it's been changed in the UI, would that suit your use case?

**Comment by trunsky** [comment]
I think that's fine - nobody should be touching it in the UI. Thank you!

**Comment by GtheSheep** [comment]
Released on v0.0.84
